### PR TITLE
Fixed Windows malformed path issue

### PR DESCRIPTION
**Pull request by benleim**
**Please refer to the contribution guidelines in the README when submitting PRs.**
Fixed issue #2305

### CODE REVIEW

File: localstack/services/awslambda/lambda_executors.py [hunk 1]
```diff
@@ -717,6 +718,13 @@ def get_host_path_for_path_in_docker(cls, path):
         return re.sub(r'^%s/(.*)$' % config.TMP_FOLDER,
                       r'%s/\1' % config.HOST_TMP_FOLDER, path)
 
+    @classmethod
+    def format_windows_path(cls, path):
+        temp = path.replace(':', '').replace('\\', '/')
```
Comment (benleim) [inline localstack/services/awslambda/lambda_executors.py:723]: Windows paths have a ':' and slashes in the wrong direction. Docker daemon won't allow this.

File: localstack/services/awslambda/lambda_executors.py [hunk 2]
```diff
@@ -717,6 +718,13 @@ def get_host_path_for_path_in_docker(cls, path):
...
+        if len(temp) >= 1 and temp[:1] != '/':
+            temp = '/' + temp
```
Comment (benleim) [inline localstack/services/awslambda/lambda_executors.py:725]: Windows paths don't begin with a slash, which is required.

File: localstack/services/awslambda/lambda_executors.py [hunk 3]
```diff
@@ -361,6 +361,7 @@ def prime_docker_container(self, runtime, func_arn, env_vars, lambda_cwd):
 
                 mount_volume = not config.LAMBDA_REMOTE_DOCKER
                 lambda_cwd_on_host = Util.get_host_path_for_path_in_docker(lambda_cwd)
+                lambda_cwd_on_host = Util.format_windows_path(lambda_cwd)
```
Comment (benleim) [inline localstack/services/awslambda/lambda_executors.py:364]: I tried to follow the format of other Util functions

**Comment by whummer** [comment]
Looks great - thanks for fixing this @benleim 👍

**Pull request merged**